src/main.zig

```diff
@@ -141,6 +143,20 @@ const Command = struct {
             else => err,
         };
 
+        // Calculate how many bytes are allocated inside `arena`.
+        // TODO This does not account for the fact that any allocations will be rounded up to the nearest page by `std.heap.page_allocator`.
+        var allocation_count: usize = 0;
+        var allocated_bytes: usize = 0;
+        {
+            var node_maybe = arena.state.buffer_list.first;
+            while (node_maybe) |node| {
+                allocation_count += 1;
+                allocated_bytes += node.data.len;
+                node_maybe = node.next;
+            }
+        }
+        log.info("Allocated {} bytes in {} regions during replica init", .{ allocated_bytes, allocation_count });
```

Comment:
When available, include the replica index in the log line (this really helps when debugging VOPR runs, since it interleaves the logs of all replicas).

```suggestion
        log.info("{}: Allocated {} bytes in {} regions during replica init", .{
            replica.replica,
            allocated_bytes,
            allocation_count,
        });
```